CGBEmu/CGBEmu.vcxproj.filters

```diff
@@ -12,18 +12,15 @@
     <ClCompile Include="source\App.cpp">
       <Filter>source</Filter>
     </ClCompile>
-    <ClCompile Include="Origem.cpp">
```

Comment:
Here I just updated the filters of the project so that we do not have references to the files we deleted